apps/src/applab/firebaseStorage.js

```diff
@@ -0,0 +1,260 @@
+'use strict';
+
+/* global Applab */
+
+var FirebaseUtils = require('./firebaseUtils');
+
+/**
+ * Namespace for Firebase storage.
+ */
+var FirebaseStorage = module.exports;
+
+function getKeysRef(channelId) {
+  var kv = FirebaseUtils.getDatabase(channelId).child('storage').child('keys');
+  return kv;
+}
+
+function getRecordsRef(channelId, tableName) {
+  return FirebaseUtils.getDatabase(channelId).child('storage').child('tables')
+    .child(tableName).child('records');
+}
```

Comment:
Does firebase not provide an easier way to access nested children? I feel like I'd want to do something like
```js
getDatabase(channelId).child(`storage/tables/${tableName}/records')
```
If not, I wonder if there would be value in adding such a helper ourselves. Perhaps the challenge is that a single node could validly be named "foo/bar"?